pcm_codec_registration.src.html

```diff
@@ -0,0 +1,117 @@
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: attribute
+        text: AudioDecoderConfig.description; url: dom-audiodecoderconfig-description
+        text: AudioDecoderConfig.sampleRate; url: dom-audiodecoderconfig-samplerate
+        text: AudioDecoderConfig.channelCount; url: dom-audiodecoderconfig-channelcount
+    type: dfn
+        for: EncodedAudioChunkType; text: key; url: dom-encodedaudiochunktype-key
+        for: EncodedAudioChunk; text: [[internal data]]; url: dom-encodedaudiochunk-internal-data-slot
+        for: EncodedAudioChunk; text: [[type]]; url: dom-encodedaudiochunk-type-slot
+        text: sample; url: sample
+        text: interleaved; url: interleaved
+        text: section on sample magnitude; url: audio-samples-magnitude
+    type: interface
+        text: EncodedAudioChunk; url: encodedaudiochunk
+        text: AudioData; url: audiodata-interface
+    type: dictionary
+        text: AudioDecoderConfig; url: dictdef-audiodecoderconfig
+        text: AudioEncoderConfig; url: dictdef-audioencoderconfig
+    type: enum
+        text: AudioSampleFormat; url: enumdef-audiosampleformat
+        text: u8; url: dom-audiosampleformat-u8
+        text: s16; url: dom-audiosampleformat-s16
+        text: s24; url: dom-audiosampleformat-s24
+        text: s32; url: dom-audiosampleformat-s32
+        text: f32; url: dom-audiosampleformat-f32
+</pre>
+
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+This codec's string begins with the prefix "pcm-", followed by a suffix that
+indicates the sample format. The complete list of strings and associated
+formats is as follows.
+
+* pcm-u8, using format {{u8}}
+* pcm-s16, using format {{s16}}
+* pcm-s24, using three [[WEBIDL#idl-octet|byte]] (24-bit) [=samples=] with
+    [=interleaved=] [[webcodecs#audio-buffer-arrangement|channel arrangement]].
+* pcm-s32, using format {{s32}}
+* pcm-f32, using format {{f32}}
+
+NOTE: [[WEBCODECS]] does not define a 24-bit {{AudioSampleFormat}}. 24-bit
+    samples are permitted within an {{EncodedAudioChunk}}, but such samples will
+    be "decoded" in {{AudioData}} objects as either {{s32}} of {{f32}}. Please
+    see [[WEBCODECS]] [=section on sample magnitude=] for addtional details.
+
+EncodedAudioChunk data {#encodedaudiochunk-data}
+================================================
+
+Linear pulse code modulation (linear PCM) describes a format where the audio
+values are sampled at a regular interval, and where the quantization levels
+between two successive values are linearly uniform.
+
+{{EncodedAudioChunk}} [=EncodedAudioChunk/[[internal data]]=] is expected to be
+a sequence of bytes of arbitrary length, with a [=sample=] occuring every N
+bits, where N is defined by the codec string. For multichannel PCM, [=samples=]
+from different channels are [=interleaved=].
```

Comment:
The fact that it's interleaved is well described here.

Reply:
Agree